src/controllers/messagesController.ts

```diff
@@ -72,7 +73,10 @@ export default class MessagesController {
           minimumId: req.query.minimum_id
           /* eslint-enable sort-keys */
         }),
-        params => this.messagesService.getMessagesByUser(user, params)
+        params =>
+          this.messageServicSelector
```

Comment:
typo

Reply:
Done in [f0c9eca](https://github.com/pagopa/io-backend/pull/882/commits/f0c9ecac61390c067788b47867ae612207dbd68c)